### PR TITLE
editor: Add single quote highlight for Vim

### PR DESCRIPTION
**Pull request by CrystalSage**
Add syntax highlighting for single quote character/byte literals. Highlight is similar to `String` type with double quotes.

### CODE REVIEW

File: editors/vim/syntax/jakt.vim [hunk 1]
```diff
@@ -79,6 +79,7 @@ syntax region jaktCommentLine start="//" end="$"
 
 
 syntax region jaktString matchgroup=jaktStringDelimiter start=+"+ skip=+\\\\\|\\"+ end=+"+ oneline contains=jaktEscape
+syntax region jaktChar matchgroup=jaktCharDelimiter start=+'+ end=+'+ oneline
```
Comment (jntrnr) [inline editors/vim/syntax/jakt.vim:82]: can this contain escapes also?

Comment (CrystalSage) [reply editors/vim/syntax/jakt.vim]: If we need it, sure! I'll add another commit.

**Comment by jntrnr** [comment]
@CrystalSage - thanks - can you squash the commits? then I think we're ready

**Comment by CrystalSage** [comment]
@jntrnr I messed up while squashing. Can I open another PR? Getting confused.

**Pull request closed without merging**